utils/init-submodules-release.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 
```

Comment:
```suggestion
function save_cd {
  cd "$@" || exit
}

```
I would put the exit check of the cd into a small function so its a) easier to write the code below without having to spell out exit and b) one cannot forget to call exit after a cd call.

If you want to make this function and future other util functions usable in all of the scripts define one util script for phasar that declares the functions and `sources` it into the other scripts.

Reply:
Thank you for the suggestions. I have created a new script as `./utils/safeCommandsSet.sh` and sourced it into other scripts which have `cd` calls. Other than the `cd` , I do not see another command which can be converted into `function`. 
In the future, other sets of `commands` can be replaced with new functions in the script.

Reply:
Thanks looks good :+1: